dmoj/cptbox/ptproc.cpp

```diff
@@ -103,6 +95,10 @@ int pt_process::protection_fault(int syscall) {
     return PTBOX_EXIT_PROTECTION;
 }
 
+void pt_process::use_seccomp(bool enabled) {
+    _use_seccomp = PTBOX_SECCOMP && enabled;
```

Comment:
This function should fail if the process has already been started, otherwise `PTRACE_O_TRACESECCOMP` "sticks" on the process, but our handling could change.